src/amo/reducers/api.js

```diff
@@ -101,9 +102,7 @@ export type SetClientAppAction = {|
 |};
 
 export function setClientApp(clientApp: string): SetClientAppAction {
-  if (!clientApp) {
-    throw new Error('clientApp cannot be falsey');
-  }
+  invariant(clientApp, 'clientApp cannot be falsey');
```

Comment:
We probably want an `invariant` in `setLang` too.

Reply:
Yes. I needed to remove a test and some code that would never have been executed to allow this, but it makes sense. I did some research and it looks like the code never should have been added, as it assumes that `lang` can be falsey but it really cannot be.